server/app/models/grid_domain_authorization.rb

```diff
@@ -9,6 +9,9 @@ class GridDomainAuthorization
   field :domain, type: String
   field :challenge, type: Hash
   field :challenge_opts, type: Hash # TODO encrypt?
+  field :authorization_type, type: String
+
+  has_one :grid_service
```

Comment:
Oh, obviously. :) yeah, not used yet for anything. Idea was to use that for the LB link so it's possible to link the SNI secret to LB when renewing.